docs/user-guide/monitoring.md

```diff
@@ -0,0 +1,161 @@
+# Prometheus and Grafana installation
+
+This tutorial will show you how to install [Prometheus](https://prometheus.io/) and [Grafana](https://grafana.com/) for the metric scraping from the NGINX Ingress controller.
+
+## Before You Begin
+
+Now, deploy the Ingress controller following the instructions [here](../deploy/index.md).
+
+Note that the yaml files that are used in this tutorial, stored in [deploy/monitoring](https://github.com/kubernetes/ingress-nginx/tree/master/deploy/monitoring) folder in GitHub repository [kubernetes/ingress-nginx](https://github.com/kubernetes/ingress-nginx).
+
+## Deploy and configure Prometheus Server
+
```

Comment:
"is needed to configure so that it can ..." -> "must be configured so that it can discover the Services endpoints."

Reply:
"running in *the* cluster"

Reply:
"no extra configuration *is* needed"